org.sonatype.m2e.webby/src/org/sonatype/m2e/webby/internal/util/PathSelector.java

```diff
@@ -57,9 +57,8 @@ private static String normalizePattern(String pattern) {
     return normalized;
   }
 
-  private static String normalizePath(String path) {
-    String normalized = path.replace((File.separatorChar == '/') ? '\\' : '/', File.separatorChar);
-    return normalized;
+  public static String normalizePath(String path) {
```

Comment:
Great, I like this approach much better! I'd like to suggest one more change though:
```java
return path.replaceAll("[\\/]", File.separatorChar);
```
It does the same thing, but is a little bit more compact.